MapboxNavigation/RoutePageViewController.swift

```diff
@@ -8,7 +8,7 @@ protocol RoutePageViewControllerDelegate: class {
     var upComingStep: RouteStep? { get }
     func stepBefore(_ step: RouteStep) -> RouteStep?
     func stepAfter(_ step: RouteStep) -> RouteStep?
-    func routePageViewController(_ controller: RoutePageViewController, willTransitionTo maneuverViewController: RouteManeuverViewController)
+    func routePageViewController(_ controller: RoutePageViewController, willTransitionTo maneuverViewController: RouteManeuverViewController, didSwipe: Bool)
```

Comment:
Is `didSwipe` supposed to indicate whether the method is being called due to a user gesture? If so, can you add a comment to that effect?